src/components/Proyectos/styles.js

```diff
@@ -37,4 +37,10 @@ export const useStyles = makeStyles((theme) => ({
     justifyContent: "flex-end",
     alignItems: "flex-end",
   },
+  buttonRoot: {
+    fontSize: "18px",
+    fontWeight: "700",
+    borderRadius: "31px",
+    padding: "7px 30px",
```

Comment:
This style seems to be duplicated. Maybe you have to put it somewhere else or create a button component that contains it.